docs/crash_reporting.md

```diff
@@ -0,0 +1,202 @@
+
+# keep the rest of the script unchanged
+
+# add these lines to compose the packager and compiler sourcemaps into one file
+REACT_NATIVE_DIR=../node_modules/react-native
+source "$REACT_NATIVE_DIR/scripts/find-node.sh"
+source "$REACT_NATIVE_DIR/scripts/node-binary.sh"
+"$NODE_BINARY" "$REACT_NATIVE_DIR/scripts/compose-source-maps.js" "$CONFIGURATION_BUILD_DIR/main.jsbundle.map" "$CONFIGURATION_BUILD_DIR/$UNLOCALIZED_RESOURCES_FOLDER_PATH/main.jsbundle.map" -o "../$SOURCEMAP_FILE"
+```
+
+To upload the source map, run this from your React Native project root:
+
+```bash
+export DATADOG_API_KEY= # fill with your API key
+export SERVICE=com.myapp # replace by your service name
+export VERSION=1.0.0 # replace by the version of your app in XCode
+export BUILD=100 # replace by the build of your app in XCode
+export BUNDLE_PATH= # fill with your bundle path
+
+yarn datadog-ci react-native upload --platform ios --service $SERVICE --bundle $BUNDLE_PATH --sourcemap ./build/main.jsbundle.map --release-version $VERSION --build-version $BUILD
+```
+
+
+### Upload JavaScript source maps on Android builds
+
+#### Manually on each build
+
+On Android, the bundle file is located at `android/app/build/generated/assets/react/release/index.android.bundle` and the source map file is located at `android/app/build/generated/sourcemaps/react/release/index.android.bundle.map`.
+
+After running your build, upload your source map by running this from your React Native project root:
+
+```bash
+export DATADOG_API_KEY= # fill with your API key
+export SERVICE=com.myapp # replace by your service name
+export VERSION=1.0.0 # replace by the versionName from android/app/build.gradle
+export BUILD=100 # replace by the versionCode from android/app/build.gradle
+export BUNDLE_PATH=android/app/build/generated/assets/react/release/index.android.bundle
+export SOURCEMAP_PATH=android/app/build/generated/sourcemaps/react/release/index.android.bundle.map
+
+yarn datadog-ci react-native upload --platform android --service $SERVICE --bundle $BUNDLE_PATH --sourcemap $SOURCEMAP_PATH --release-version $VERSION --build-version $BUILD
+```
+
+
+### Upload iOS dSYM files
+
+#### Manually on each build
+
+For more information, see the [iOS Crash Reporting and Error Tracking documentation][4].
+
+
+### Upload Android Proguard mapping files
+
+First, ensure that Proguard minification is enabled on your project. By default, this is not enabled on React Native projects.
+
+For more information, see [the React Native Proguard documentation][5].
+
+If you are still unsure, you can see if running `(cd android && ./gradlew tasks --all) | grep minifyReleaseWithR8` returns anything. If so, minification is enabled.
+
+#### Manually on each build
+
+In your `android/app/build.gradle` file, add the plugin and configure it **at the very top of the file**:
+
+```groovy
+plugins {
+    id("com.datadoghq.dd-sdk-android-gradle-plugin") version "1.4.0"
+}
+
+datadog {
+    site = "US"
```

Comment:
it is the legacy name, better to use "US1", etc.

Also I think we should add a link to the Gradle plugin docs, just in case, and also mention that value of the side should match the value of the side in the SDK configuration.

Reply:
The link in Line 170 sends users to the Android Gradle Plugin page.

Reply:
I'd rather not put a value that is not correct inside a code snippet, especially in native code that people might not be familiar with.
Let's put `US1` here and mention the other sites below.

```suggestion
    site = "US1"
```